src/main/java/pl/cleankod/exchange/core/service/AccountService.java

```diff
@@ -0,0 +1,13 @@
+package pl.cleankod.exchange.core.service;
+
+import pl.cleankod.exchange.core.domain.Account;
+import pl.cleankod.exchange.entrypoint.model.AccountDto;
+
+
+
+public interface AccountService {
+
+    AccountDto findAccountById(Account.Id accountId, String currency);
```

Comment:
Why did you return the `AccountDto` object here instead of the `Account` object?